interface/src/app/node/enriched_genesets.tpl.html

```diff
@@ -0,0 +1,21 @@
+<h4>
+  <b>Enriched gene sets: </b>
+</h4>
+
+<table id="enrichment-table" class="table table-striped table-bordered table-condensed">
+  <thead>
+    <tr>
+      <td>Pathways/Processes/Diseases</td>
+      <td>Databases</td>
+      <td>P-Value</td>
+      <td>Genes</td>
+    </tr>
+  </thead>
+
+  <tr ng-repeat="geneset in enrichedGenesets">
+    <td><a href="{{geneset.url}}">{{geneset.name}}</a></td>
+    <td>{{geneset.dbase}}</td>
+    <td>{{geneset.pValue}}</td>
+    <td><span ng-repeat="gene in geneset.genes"> {{gene.stdName}} </span></td>
```

Comment:
@mhuyck and @dongbohu: That makes sense - I am putting in Matt's suggested implementation, and that way, if we decide to use comma and space characters to separate the genes in the future, we can just add it to the function. Thanks!